store.go

```diff
@@ -2808,11 +3023,22 @@ func (s *store) LayerParentOwners(id string) ([]int, []int, error) {
 	if rlstore.Exists(id) {
 		return rlstore.ParentOwners(id)
 	}
+
+	if crlstore != nil {
```

Comment:
There are enough instances of the
```
primary.startReading()
code(primary)
if containerLayers != nil  {
   containerLayers.startReading()
   code(containerLayers) 
}
```

that the duplication starts being worrisome. Even the simplest copy & paste job gets typos at a large enough scale, and future updates that forget to update one of the paths.

Introduce a `readWritableLayerStores` helper, please.